app/models.py

```diff
@@ -211,9 +211,9 @@ class Member_Steps(db.Model):
 '''
 class Member_Proofs(db.Model):
     proof_name = db.Column(db.String(64), db.ForeignKey('proof.proof_name'), primary_key=True)
-    username = db.Column(db.String(64), db.ForeignKey('member.username'))
+    username = db.Column(db.String(64), db.ForeignKey('member.username'), primary_key=True)
     proof_verified_by = db.Column(db.String(64))
-    proof_document = db.Column(db.String(64), db.ForeignKey('proof.proof_document'), primary_key=True)
+    proof_document = db.Column(db.String(64))
```

Comment:
Maybe I believe this proof_doc column no longer needs to be here. Proof_name=proof_doc is my understanding. Is that wrong?

Reply:
resolved